--- libhubbub-0.2.0/Makefile~	2013-04-19 21:48:41.000000000 +0300
+++ libhubbub-0.2.0/Makefile	2013-04-27 17:02:07.366422833 +0300
@@ -61,6 +61,7 @@
 endif
 
 # Extra installation rules
+lib := lib
 I := /include/hubbub
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):include/hubbub/errors.h
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):include/hubbub/functypes.h
@@ -68,5 +69,5 @@
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):include/hubbub/parser.h
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):include/hubbub/tree.h
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):include/hubbub/types.h
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib/pkgconfig:lib$(COMPONENT).pc.in
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib:$(OUTPUT)
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib)/pkgconfig:lib$(COMPONENT).pc.in
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib):$(OUTPUT)
